components/VoteTicker/VoteTicker.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect } from "react";
+import styled from "styled-components";
+import Clock from "public/assets/clock.svg";
+import UpRightArrow from "public/assets/up-right-arrow.svg";
+import calculateTimeRemaining from "./calculateTimeRemaining";
+
+type theme = "light" | "dark";
+
+interface Props {
+  theme: theme;
+}
+
+const VoteTicker: React.FC<Props> = ({ theme }) => {
+  const { timeRemaining } = useVoteTicker();
+  return (
+    <Section theme={theme}>
+      <Wrapper theme={theme}>
+        <VoteBlock theme={theme}>
+          <ClockBG theme={theme}>
+            <Clock />
+          </ClockBG>
+          <VoteText theme={theme}>
+            Time to commit vote:
```

Comment:
this probably should be a  prop

Reply:
Yeah fair. I guess we need two props here:

1) Enum for commit reveal (similar to return from Contract)
2) Num Votes (determines if this even renders).

WDYT?

Reply:
id put the condition of rendering into the parent, and probably just pass a string prop for the label, it doesnt need to be strongly typed imo, but if u want to make an enum thats fine